React-Projects/job-portal/src/MyApplications.jsx

```diff
@@ -0,0 +1,88 @@
+import React, { useState, useEffect } from "react";
+import "./styles/MyApplications.scss";
+import { fetchMyApplications } from "./api.js";
+import { useNavigate } from "react-router-dom";
+import lodash from "lodash";
+import { JWT_COOKIE } from "./Constants.jsx";
+import { toast } from "react-toastify";
+import noJobLogo from "./assets/people.png";
+
+const MyApplicationCard = ({ myApplication }) => {
+  return (
+    <li>
+      <div className="application">
+        <div className="application-container">
+          <div className="application-info">
+            <h2>{myApplication.applicant}</h2>
+            <p>
+              <strong>Cover letter:</strong> {myApplication.cover_letter}
+            </p>
+            <p>
+              <strong>Job title:</strong> {myApplication.job_title}
+            </p>
+            <p>
+              <strong>Applied at:</strong> {`${myApplication.applied_at}`}
+            </p>
+          </div>
+          <div
+            className="status-container"
+            style={{
+              backgroundColor:
+                myApplication.is_selected == "ACCEPTED"
+                  ? "green"
+                  : myApplication.is_selected == "REJECTED"
+                  ? "red"
+                  : "orange",
```

Comment:
@codal-kcolaco avoid using ? : (triary op) inside other ?: op. Find other way to solve it.